### PR TITLE
Create technical-content-specialist.md

### PR DESCRIPTION
**Pull request by jmunroe**

### CODE REVIEW

**Comment by jmunroe** [comment]
Waiting on internal approvals and CS&S approval before posting.

**Comment by jmunroe** [comment]
This post has received approval from CS&S to proceed.

**Comment by jmunroe** [comment]
@choldgraf Am I ok to merge this?

File: content/jobs/2023/technical-content-specialist.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+* Demonstrated excellence in communicating complex technical information to learners with a different levels of background and experience
+* Demonstrated experience in using open-science tools (e.g. Python or R) in a scientific or scholarly domain
+* Willingness to continually learn and share emerging technologies and techniques in the interactive computing and open science ecosystems
+
+### Useful qualities
+* Experience with open source workflows, research and educational contexts, and an understanding of the value that cloud-based infrastructure provides
+* Experience with the Jupyter ecosystem and other tools for interactive computing
+* Experience working with distributed service teams that use asynchronous methods of communication
+* Experience collaborating and coordinating work via online platforms and distributed revision control
+* Ability to communicate in Spanish, Portuguese, or French
```
Comment (damianavila) [inline content/jobs/2023/technical-content-specialist.md:39]: communicate = verbally and written?
Just curious...

File: content/jobs/2023/technical-content-specialist.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+* Authoring content for 2i2c for use in marketing, blog posts, and website copy.
+
+
+## Our benefits and compensation
+We believe it is important for mission-driven non-profits to also provide competitive benefits and compensation packages. See [our compensation philosophy page](https://2i2c.org/jobs/#our-compensation-philosophy) for more information about our compensation and benefits.
+
+## How to apply
+2i2c is committed to hiring processes that are inclusive of people with many lived experiences and qualities. We try to structure our hiring process so that it is predictable, doesn’t take too long, and doesn’t take too much effort.
+
+If you’d like to apply for this position, please upload your resume and cover letter using this [application form](https://forms.gle/H43CsVUoF7e2C2Nm8). We will begin reviewing applications after 2023-08-02 and will fill this position on a rolling basis once we find a candidate with the right fit for the job.
```
Comment (damianavila) [inline content/jobs/2023/technical-content-specialist.md:60]: I like the google form structure!

**Review by damianavila — COMMENTED** [review]
LGTM, but since I am not on the hiring committee I would defer the approval to someone else from the team in that committee.

File: content/jobs/2023/technical-content-specialist.md (hunk 1 above)
Comment (jmunroe) [reply content/jobs/2023/technical-content-specialist.md:39]: Both have value!

**Comment by jmunroe** [comment]
Thanks @damianavila  - -the job description has already gone through the the hiring committee. My request for a review was primarily that I was doing the Hugo stuff correctly.

Giving your check, I'll merge now.

**Comment by damianavila** [comment]
> My request for a review was primarily that I was doing the Hugo stuff correctly.

Next time, it would be useful to state that in the first/top message because otherwise, people will just focus on the pieces they believe is important, and I actually focused on the content 😜 instead of the Hugo machinery.

**Comment by jmunroe** [comment]
I appreciate the review though! It wasn't *just* for the Hugo stuff but I'll try to be more clear in the future.

**Pull request merged**